kolibri/utils/options.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+
+from configobj import ConfigObj
+from configobj import flatten_errors
+from configobj import get_extra_values
+from validate import Validator
+
+
+logger = logging.getLogger(__name__)
+
+ini_specification = [
+
+    "[Database]",
+    "DATABASE_ENGINE = option('sqlite', 'postgres', default='sqlite')",
+    "DATABASE_NAME = string(default='')",
+    "DATABASE_PASSWORD = string(default='')",
+    "DATABASE_USER = string(default='')",
+    "DATABASE_HOST = string(default='localhost')",
+
+    "[Paths]",  # these are defined relative to KOLIBRI_HOME (or absolute)
+    "CONTENT_ROOT = string(default=content)",
```

Comment:
we could choose a conventional way to name variables that hold paths. E.g. `KOLIBRI_HOME_DIR`, `CONTENT_DIR`, etc

Reply:
Yeah, happy to change this name if desired. If we change `KOLIBRI_HOME` to `KOLIBRI_HOME_DIR`, though, it could be a headache in terms of maintaining support for both the old name and the new name?

(The `_DIR` convention is something we've been using in our Django settings files -- the `_ROOT` suffix is a Django convention, e.g. for `STATIC_ROOT` and `MEDIA_ROOT`, and indicates it's a base directory under which there are subdirectories with their own structure)